src/nibbler.js

```diff
@@ -23,6 +23,29 @@ function getCounts(messages) {
   return counts;
 }
 
+// Extracted from Eslint https://github.com/eslint/eslint/blob/ba58d94cb51d4d2644c024446d5750eaf4853129/lib/cli-engine/cli-engine.js#L181-L202
```

Comment:
Ah cool, nice find.  I was thinking we'd have to do something like this.  Thanks for the note pointing to the code in ESLint.